### PR TITLE
Fixing public Jira issue : APIMANAGER-2703

### PR DESCRIPTION
**Pull request by ChamNDeSilva**
Fixing - Blank drop down menu and wrong caption in the absence of tier permission for the logged in user.

### CODE REVIEW

File: features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store/site/themes/fancy/templates/api/api-info/template.jag [hunk 1]
```diff
@@ -202,13 +206,12 @@ if(tenant!=null) {
...
                         <%
-                            var tierArr = api.tiers;
                             var firstTierDesc = '';
-                            if(tierArr != null){
-                                firstTierDesc = tierArr[0].tierDescription;
-                                if(firstTierDesc == "null"){
-                                    firstTierDesc='';
-                                }
+                            if(existingFirstTierDesc.length ==0){
+                                firstTierDesc = '';
```
Comment (harsha89) [inline features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store/site/themes/fancy/templates/api/api-info/template.jag]: follow 
```
if () {
//logic
} else { //no space after }
//logic
}
```

File: features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store/site/themes/fancy/subthemes/tabs/templates/api/api-info/template.jag [hunk 2]
```diff
@@ -182,8 +185,13 @@
                                                                     denied = false;
                                                                      } %>
                                                                 </select>
-                                                                <%var firstTierDesc=api.tiers[0].tierDescription;
-                                                                if(firstTierDesc=="null"){firstTierDesc='';}
+                                                                <% var firstTierDesc = '';
+                                                                if(existingFirstTierDesc.length==0){
+                                                                   firstTierDesc = '';
+                                                                }
```
Comment (harsha89) [inline features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store/site/themes/fancy/subthemes/tabs/templates/api/api-info/template.jag]:
```
if () {
//logic
} else { //no space after }
//logic
}
```

Comment (ChamNDeSilva) [inline features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store/site/themes/fancy/subthemes/tabs/templates/api/api-info/template.jag]: Fixed in PR https://github.com/wso2/carbon-apimgt/pull/1093

File: features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store/site/themes/fancy/templates/api/api-info/template.jag (hunk 1 above)
Comment (ChamNDeSilva) [inline features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store/site/themes/fancy/templates/api/api-info/template.jag]: Fixed in PR https://github.com/wso2/carbon-apimgt/pull/1093

**Pull request merged**